src/dhtproto/client/UsageExamples.d

```diff
@@ -323,10 +344,18 @@ unittest
                 case node_error:
                 case wrong_node:
                 case unsupported:
+                case timed_out:
```

Comment:
Nice, so it has immediately found a bug? :)

Reply:
Yes, indeed :). Yes, I'll move dhtproto to a newer compiler.